.travis.yml

```diff
@@ -39,6 +39,10 @@ script:
       mvn clean install -B -U
     fi
 
+# enable codecov-io
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
This seems to be the recommended way to integrate it. We do have to trust them just like we trust Travis.